frontend/src/ManageOrgUsers/InviteUsersForm.jsx

```diff
@@ -109,7 +111,7 @@ function InviteUsersForm({
             <div className="card-header-inner-wrap">
               <h3 className="tj-text-lg tj-text font-weight-500" data-cy="add-users-card-title">
                 {!isEditing
-                  ? t('header.organization.menus.manageUsers.addNewUser', 'Add new user')
+                  ? t('header.organization.menus.manageUsers.inviteNewUsers', 'Add new user')
```

Comment:
Could you please revert this change? The related GitHub issue doesn't have this change requested.